r/R/dplyr-funcs-datetime.R

```diff
@@ -300,6 +312,34 @@ register_bindings_duration <- function() {
 
     build_expr("cast", x, options = cast_options(to_type = duration(unit = "s")))
   })
+  register_binding("decimal_date", function(date) {
+    # browser()
+    y <- build_expr("year", date)
+    # timezone <- call_binding("tz", date)
+    start <- call_binding("make_datetime", year = y, tz = "UTC")
+    end <- call_binding("make_datetime", year = y + 1L, tz = "UTC")
+    # maybe use yday here
```

Comment:
I think using `difftime` might be a bit more robust since the number of seconds (not days) from the beginning of the year would give us a more precise decimal representation (and that is what lubridate does).